phase1/vsphere/Kconfig

```diff
@@ -31,36 +31,36 @@ config phase1.vSphere.datacenter
 	string "Datacenter"
 	default "datacenter"
 	help
-		The datacenter to be used by vSphere cloud provider plugin.
+		The datacenter to be used by vSphere cloud provider plugin. Use vCenter UI or ```govc datacenter.info```
 
 config phase1.vSphere.datastore
 	string "Datastore"
 	default "datastore"
 	help
-		The datastore where VMs will be created.
+		The datastore where VMs will be created. This will also be used for provisioning volumes for storage classes.
 
 config phase1.vSphere.resourcepool
-        string "Resourcepool"
-        default "/datacenter/host/<host-ip> (or) <vc-cluster-name>/Resources"
+        string "Specify a valid Cluster, Host or Resource Pool in which to deploy Kubernetes VMs."
+        default ""
         help
-                The resourcepool where VMs will be created.
+		Specify a valid Cluster, Host or Resource Pool in which to deploy Kubernetes VMs.
```

Comment:
May be change it to "Specify a valid Cluster, Host or Resource Pool where Template was uploaded. Kubernetes VMs will be provisioned in same Cluster/Host/RP".